org.osgi.test.assertj.cm/src/main/java/org/osgi/test/assertj/cm/configurationevent/ConfigurationEventConditions.java

```diff
@@ -0,0 +1,294 @@
+
+	/**
+	 * Creates a {@link Condition} to be met by an <code>ConfigurationEvent.
+	 * Checking if a given pid <b>matches</b> the pid of the Event.
+	 *
+	 * @param pid the pid
+	 * @return the condition
+	 */
+	public static Condition<ConfigurationEvent> pid(final String pid) {
+
+		return VerboseCondition.verboseCondition((ConfigurationEvent e) -> Objects.equals(e.getPid(), pid),
+			"pid equals <" + pid + ">", //
+			e -> " was <" + e.getPid() + ">");
+
+	}
+
+	/**
+	 * Creates a {@link Condition} to be met by an <code>ConfigurationEvent.
+	 * Checking if a given factoryPid <b>matches</b> the factoryPid of the
+	 * Event.
+	 *
+	 * @param factoryPid the factory pid
+	 * @return the condition
+	 */
+	public static Condition<ConfigurationEvent> factoryPid(final String factoryPid) {
+
+		return VerboseCondition.verboseCondition(
+			(ConfigurationEvent e) -> Objects.equals(e.getFactoryPid(), factoryPid),
+			"factoryPid equals <" + factoryPid + ">", //
+			e -> " was <" + e.getFactoryPid() + ">");
+
+	}
+
+	/**
+	 * Creates a {@link Condition} to be met by an <code>ConfigurationEvent.
+	 * Checking if the type <code>ConfigurationEvent.CM_DELETED</code>
+	 * <b>matches</b> the type of the Event.
+	 *
+	 * @return the condition
+	 */
+	public static Condition<ConfigurationEvent> typeDeleted() {
+		return type(ConfigurationEvent.CM_DELETED);
+	}
+
+	/**
+	 * Creates a {@link Condition} to be met by an <code>ConfigurationEvent.
+	 * Checking if the type <code>ConfigurationEvent.CM_LOCATION_CHANGED</code>}
+	 * <b>matches</b> the type of the Event.
+	 *
+	 * @return the condition
+	 */
+	public static Condition<ConfigurationEvent> typeLocationChanged() {
+		return type(ConfigurationEvent.CM_LOCATION_CHANGED);
+	}
+
+	/**
+	 * Creates a {@link Condition} to be met by an <code>ConfigurationEvent.
+	 * Checking if the type <code>ConfigurationEvent.CM_UPDATED</code>}
+	 * <b>matches</b> the type of the Event.
+	 *
+	 * @return the condition
+	 */
+	public static Condition<ConfigurationEvent> typeUpdated() {
+		return type(ConfigurationEvent.CM_UPDATED);
+	}
+
+	/**
+	 * Creates a {@link Condition} to be met by an <code>ConfigurationEvent.
+	 * Checking if a given pid <b>matches</b> the pid and a given type
+	 * <b>matches</b> the type of the Event.
+	 *
+	 * &#64;param eventType the type that would be checked against the type of the
+	 *            <code>ConfigurationEvent</code>}
+	 * 
+	 * @param pid the pid
+	 * @return the condition
+	 */
+	public static Condition<ConfigurationEvent> typeAndPid(final int eventType, String pid) {
+		return allOf(type(eventType), pid(pid));
+	}
+
+	/**
+	 * Creates a {@link Condition} to be met by an <code>ConfigurationEvent.
+	 * Checking if a given pid <b>matches</b> the pid and the type
+	 * <code>ConfigurationEvent.CM_DELETED</code> <b>matches</b> the type of the
+	 * Event.
+	 *
+	 * @param pid the pid
+	 * @return the condition
+	 */
+	public static Condition<ConfigurationEvent> typeDeletedAndPid(String pid) {
```

Comment:
We don't need method with type names in them.

Reply:
removed